kahuna/public/js/search/index.js

```diff
@@ -267,4 +267,15 @@ search.run(['$rootScope', '$state', function($rootScope, $state) {
             $state.go('search.results', null, {reload: true});
         }
     });
+    $rootScope.$on('$stateChangeStart', (_, toState, toParams) => {
+        const collectionQuery = toParams.query && toParams.query.indexOf('~') === 0;
+        if (toState.name === 'search.results' && collectionQuery) {
+            toParams.orderBy = 'dateAddedToCollection';
+        }
+        else {
```

Comment:
Shouldn't this check the `toState.name` too? Might be worth nesting the whole thing within that condition, since it's the scope required for this search controller.